pom.xml

```diff
@@ -120,22 +125,7 @@
         <dependency>
             <groupId>javax.xml.bind</groupId>
             <artifactId>jaxb-api</artifactId>
-            <version>2.2.11</version>
```

Comment:
does this change go away when https://github.com/dsmHack/2018_team_1_api/pull/39 merges?